src/WebJobs.Script/Diagnostics/DiagnosticEventConstants.cs

```diff
@@ -13,5 +13,8 @@ internal static class DiagnosticEventConstants
 
         public const string SasTokenExpiringErrorCode = "AZFD0006";
         public const string SasTokenExpiringErrorHelpLink = "https://go.microsoft.com/fwlink/?linkid=2244092";
+
+        public const string UnsupportedZipDeploymentOnLinuxConsumptionwithMSIErrorCode = "AZFD0007";
+        public const string UnsupportedZipDeploymentOnLinuxConsumptionwithMSIErrorCodeLink = "<ToDo>";
```

Comment:
ToDo - Add document in [azure-docs repo](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/azure-functions/errors-diagnostics/diagnostic-events/azfd0006.md) and link it here before merging